Wikipedia/Code/ReadingListHintViewController.swift

```diff
@@ -13,7 +13,14 @@ class ReadingListHintViewController: UIViewController {
     }
     
     private var hintButtonTitle: String {
-        let articleTitle = article?.displayTitle ?? "article"
+        var articleTitle = "article"
```

Comment:
It'd be nice if this wasn't hardcoded to "article" in English, but that's outside the scope of this PR

Reply:
(should be nil or empty by default, then have a separate localized string that is "Add this article to a reading list?" that shows when the title is nil or empty)